sdk/purview/azure-analytics-purview-catalog/swagger/README.md

```diff
@@ -1,6 +1,6 @@
 ## Generate autorest code
 ```yaml
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2021-05-01-preview/purviewcatalog.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/ccbe894f6b012ca2000184307ed453fd68797b86/specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2022-03-01-preview/purviewcatalog.json
```

Comment:
why do we use feature branch instead of main branch? why not merge the swagger spec pr ?